src/Microsoft.Health.Dicom.Core.UnitTests/Features/Query/QueryParserTests.cs

```diff
@@ -129,13 +129,25 @@ public void GivenFilterCondition_ValidReferringPhysicianNameTag_CheckProperties(
     }
 
     [Theory]
-    [InlineData("00080061", "CT")]
+    [InlineData("00201208", "3")]
     public void GivenFilterCondition_WithNotSupportedTag_Throws(string key, string value)
     {
         Assert.Throws<QueryParseException>(() => _queryParser
             .Parse(CreateParameters(GetSingleton(key, value), QueryResource.AllStudies), QueryTagService.CoreQueryTags));
     }
 
+    [Theory]
+    [InlineData("ModalitiesInStudy", "CT")]
+    public void GivenFilterCondition_WithModalitiesInStudySupportedTag_Works(string key, string value)
```

Comment:
Should we add one test for new include fields?

Reply:
It is covered in E2E test